cmd/tealdbg/local.go

```diff
@@ -90,12 +90,8 @@ func txnGroupFromParams(dp *DebugParams) (txnGroup []transactions.SignedTxn, err
 
 	// if conversion failed report all intermediate decoding errors
 	if err != nil {
-		if err1 != nil {
-			log.Printf("Decoding as JSON txn failed: %s", err1.Error())
-		}
-		if err2 != nil {
-			log.Printf("Decoding as JSON txn group failed: %s", err2.Error())
-		}
+		log.Printf("Decoding as JSON txn failed: %v", err1)
```

Comment:
these err1 and err2 can be actually be nil and should not reported (here and below)

Reply:
yes, the idea is to try different format as bail out as soon as something successfully decoded.
`err` is not logged because it is returned and logged somewhere downstream

Reply:
oh, that's actually right, both must be non-nil when reaching this point.